apps/angular/src/app/features/anime/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { Anime } from '@js-camp/core/models/anime';
+import { AnimeListQueryOptions } from '@js-camp/core/models/animeListQueryOptions';
+import { Pagination } from '@js-camp/core/models/pagination';
+import { Sorting, SortTitle, SortValue } from '@js-camp/core/models/sorting';
+import { Observable } from 'rxjs';
+
+import { DEFAULT_ACTIVE_PAGE, DEFAULT_LIMIT, DEFAULT_OFFSET, DEFAULT_TOTAL_PAGE } from '../../../../constants';
+
+import { AnimeService } from '../../../../core/services/anime.service';
+
+/** Anime table list. */
+@Component({
+  selector: 'camp-anime-table',
+  templateUrl: './anime-table.component.html',
+  styleUrls: ['./anime-table.component.css'],
+})
+export class AnimeTableComponent {
+
+  /** Pagination result. */
+  public readonly result$: Observable<Pagination<Anime>>;
+
+  /** Default query options of anime list. */
+  public defaultQuery = new AnimeListQueryOptions({
+    limit: DEFAULT_LIMIT,
+    offset: DEFAULT_OFFSET,
+    activePage: DEFAULT_ACTIVE_PAGE,
+    totalPages: DEFAULT_TOTAL_PAGE,
+    sorting: new Sorting({
+      title: SortTitle.TitleEnglish,
+      value: SortValue.TitleEnglish,
+      isAscending: true,
+    }),
+  });
+
+  public constructor(private animeService: AnimeService) {
+    this.result$ = this.animeService.getAnimeList(this.defaultQuery);
+  }
```

Comment:
If you are not going to use `animeService` outside of constructor, then you can omit access modifier 
```suggestion
  public constructor(animeService: AnimeService) {
    this.result$ = animeService.getAnimeList(this.defaultQuery);
  }
```
Otherwise, please add readonly
```suggestion
  public constructor(private readonly animeService: AnimeService) {
    this.result$ = this.animeService.getAnimeList(this.defaultQuery);
  }
```